eclipse/src/saros/ui/util/SWTUtils.java

```diff
@@ -187,7 +187,12 @@ public static void runSafeSWTSync(final Logger log, final Runnable runnable) {
    * @return the display of the current workbench
    */
   public static Display getDisplay() {
-    return PlatformUI.getWorkbench().getDisplay();
+    // return PlatformUI.getWorkbench().getDisplay();
```

Comment:
There is a reason we do not use` Display#getDefault`. 

When Eclipse shuts down it disposes the display. With this change you will creating a new one and this is something we do not want to happen.

Reply:
So after looking at the issue this is not the correct fix.

Either we ensure that the Platform is available when our context is started or we modify the affected classes.

Reply:
I pushed a new Commit that contains a fix by keeping track of the display after its first creation.
But it should be seen as a temporary fix that we chose because of time constraints.
The better solution would probably be to change all affected classes since there doesnt seem to be an equivalent to  PlatformUI.getWorkbench().getDisplay() in E4.

Reply:
Looking at the issue, well I do not know. I could modify the affected class. Regarding your solution and looking at the current call stack I guess it would be sufficient just to call getCurrent (assuming the Display was already created and Eclipse call us on the creator thread). Of course your solution works (if the display was created, even if there is no workbench) but it requires good knowledge on SWT / Eclipse Startup.